### PR TITLE
Feature/update operation guildes

### PR DESCRIPTION
**Pull request by kgalli**
Our operation guides are missing information on how to perform backups and how to perform upgrades to new OpenProject versions. This PR is going to fill this gap step by step.

https://community.openproject.org/work_packages/19492

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://kgalli-feature-update-operation-guildes.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://kgalli-feature-update-operation-guildes.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: doc/operation_guides/manual/backup-guide.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+```
+*Note:* You can restore any database backup with the following
+command. Be aware that you have to replace the
+`/path/to/backup/file.backup` path with your actual backup path.
+
+```bash
+RAILS_ENV=production bundle exec rake backup:database:restore[/path/to/backup/file.backup]
+```
+
+If your database dump is from an old version of OpenProject, also run
```
Comment (NobodysNightmare) [inline doc/operation_guides/manual/backup-guide.md:39]: "also run" does not indicate the timing very well.

I assume that you should restore first and migrate afterwards. Better make that clear, e.g.

> If your database dump is from an old version of OpenProject, also run the following command after restoring:

File: doc/operation_guides/manual/upgrade-guide.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+* If you run the worker process with a cronjob, disable the cronjob temporarily.
+* Stop the (delayed\_job) worker process. In case you run the woker process through
+  `RAILS_ENV=production bundle exec script/delayed_job start`, execute the following:
+  `RAILS_ENV=production bundle exec script/delayed_job stop`.
+
+## Update your system
+
+[root@debian]# apt-get update
+[root@debian]# apt-get upgrade
```
Comment (NobodysNightmare) [inline doc/operation_guides/manual/upgrade-guide.md:30]: I think you forgot to mark that section as code. Is is displayed oddly as rendered markdown.

**Comment by NobodysNightmare** [comment]
looking good so far,
tell me when you are done ^^

**Comment by kgalli** [comment]
@NobodysNightmare I changed things in regard to your comments. I also did some minor updates to the installation guide.

**Comment by jonasheinrich** [comment]
:+1:

**Pull request merged**